test/testdata/dsl/rails/delegate.rb

```diff
@@ -45,3 +45,11 @@ class IgnoredUsages
   delegate :foo, :bar, to: :@hi, prefix: true # error: Method `delegate` does not exist
   delegate :foo, :bar, to: '', prefix: true # error: Method `delegate` does not exist
 end
+
+class EnumerableUsage
+  include Enumerable
+
+  Elem = type_member
+
+  delegate :each, to: :foo
```

Comment:
I think this will fail with `# error: Method `delegate` does not exist`

Reply:
Hmm, it doesn't and I don't see why it should. After all `delegate` is a built-in DSL pass which gets triggered in when any class method named `delegate` is encountered, isn't it?

Reply:
Ack, I got confused by `IgnoredUsages` part that errors for good reason